ccmlib/dse/dse_node.py

```diff
@@ -57,7 +57,13 @@ def get_version_from_build(install_dir=None, node_path=None, cassandra=False):
                 else:
                     return LooseVersion(dse_version)
             # Source cassandra installs we can read from build.xml
-            return Node.get_version_from_build(install_dir, cassandra)
+            build = os.path.join(install_dir, 'build.xml')
```

Comment:
Are we ever going to hit this for a dse node? Shouldn't it be removed altogether?